### PR TITLE
Add support for PaaS

### PR DESCRIPTION
**Pull request by bogdal**
- [x] configuration stored in environment variables
 - [x] support for serving static files
 - [x] additional dependencies

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 *.py[cod]
-.*
+.env
```
Comment (patrys) [inline .gitignore]: What is `.env` and why is it special? We should be ingoring all dot-files with some exceptions. I don't want to have to list all possible IDE/environment cruft here.

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,7 @@ dist/
 #data
 /media/
 /static/
+/node_modules/
+.idea
+saleor/static/css/dashboard.css
```
Comment (patrys) [inline .gitignore]: Are there `.css` files we actually want to include?

File: saleor/settings.py [hunk 3]
```diff
@@ -57,7 +66,7 @@
 ]
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '{{ secret_key }}'
+SECRET_KEY = os.environ.get('SECRET_KEY')
```
Comment (patrys) [inline saleor/settings.py]: How about `os.environ.get('SECRET_KEY', '{{ secret_key }}')`? This way using this as a Django project template will continue to work.

Comment (bogdal) [inline saleor/settings.py]: Yeah, I totally agree with you. Thanks!

**Pull request merged**